workers/unity/Assets/Playground/Scripts/Worker/SteamClientWorkerConnector.cs

```diff
@@ -0,0 +1,123 @@
+        ///     authenticating a Steam user trying to join a SpatialOS deployment using Steamworks.NET.
+        /// </summary>
+        /// <param name="steamTicket">Steam auth session token in case true is returned.</param>
+        /// <param name="errorMessage">Reason for failing the retrieval in case false is returned.</param>
+        /// <returns>True, if a Steam auth session token was successfully retrieved.</returns>
+        private bool GetSteamAuthSessionTicketSteamworksNET(out string steamTicket, out string errorMessage)
+        {
+            steamTicket = string.Empty;
+            errorMessage = "Method not implemented.";
+            return false;
+
+            // Uncomment this code and remove the preceding method stub after installing Steamworks.NET. (https://steamworks.github.io)
+            /*
+            steamTicket = string.Empty;
+            errorMessage = string.Empty;
+            if (!SteamManager.Initialized)
+            {
+                errorMessage = "SteamManager is not initialized. " +
+                    "Ensure that you have a SteamManager MonoBehaviour instance in the scene and that this code is not called before SteamManager.Awake(). " +
+                    "Also ensure that your Steam client is running and that you are logged in.";
+                return false;
+            }
+
+            var steamTicketData = new byte[1024];
+            var hAuthTicket = Steamworks.SteamUser.GetAuthSessionTicket(steamTicketData, 1024, out var steamTicketLength);
+            if (hAuthTicket == HAuthTicket.Invalid)
+            {
+                errorMessage = "SteamUser.GetAuthSessionTicket returned with invalid result.";
+                return false;
+            }
+
+            steamTicket = System.BitConverter.ToString(steamTicketData, 0, (int) steamTicketLength).Replace("-", "");
+            return true;
+            */
+        }
+
+        /// <summary>
+        ///     Retrieves a Steam auth session token (referred to as Steam ticket in the SpatialOS worker SDK) for
+        ///     authenticating a Steam user trying to join a SpatialOS deployment using Facepunch.Steamworks.
+        /// </summary>
+        /// <param name="steamTicket">Steam auth session token in case true is returned.</param>
+        /// <param name="errorMessage">Reason for failing the retrieval in case false is returned.</param>
+        /// <returns>True, if a Steam auth session token was successfully retrieved.</returns>
+        private bool GetSteamAuthSessionTicketFacepunch(out string steamTicket, out string errorMessage)
+        {
+            steamTicket = string.Empty;
+            errorMessage = "Method not implemented.";
+            return false;
+
+            // Uncomment this code and remove the preceding method stub after installing Facepunch.Steamworks. (https://github.com/Facepunch/Facepunch.Steamworks)
+            /*
+            if (Facepunch.Steamworks.Client.Instance == null)
```

Comment:
This is WIP and isn't really working yet as we depend on the Start() function for initialising the steam client being called before the Start() function of our WorkerConnector which attempts to connect. Just wanted to save my progress over the weekend.